odra-schema/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "odra-schema"
+edition = "2021"
+version.workspace = true
+authors.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+
+[dependencies]
+casper-contract-schema = { git = "https://github.com/odradev/casper-contract-schema" }
```

Comment:
We should release this to crates.io

Reply:
You hold the keys. :)

Reply:
Let's merge it as is, and I'll review and release `casper-contract-schema` and then make new PR with change.